libs/sdk-backend-base/src/dummyBackend/index.ts

```diff
@@ -736,6 +738,14 @@ class DummyWorkspaceSettingsService implements IWorkspaceSettingsService {
     setLocale(_locale: string): Promise<void> {
         return Promise.resolve();
     }
+
+    setTimeZone(_timeZoneId: string): Promise<void> {
```

Comment:
- Should we change the function name `setTimeZone` → `setTimezone`?
  `Timezone` itself is a valid word, there is no need to capitalize the letter `Z`, is there?

- Should we change the param name `_timeZoneId` → `_timezone`?